### DISCUSSION TITLE
Hide Link Icons / Örk Simgelerini Gizle

### DISCUSSION THREAD

**Discussion by volkantash** [GitbookIO/community#50] [Feature Requests]
### What's your feature/idea?

I would like an option to hide the icons of the pages when adding links. When I add a link to a page between the article, it is not a good option to have an icon appear at the beginning. I should be able to choose other skins wherever I want.

Yes. Add a link to the article. I sense this. These posts will not change when I change the title of the page in the future. Since it's not dynamic, I'll have to find and update all the link posts. It's not useful.

---

Bağlantı eklerken sayfaların simgelerini gizleme seçeneği istiyorum. Yazı arasında bir sayfanın linkini eklediğimde başında simge görünmesi hiç güzel bir seçenek değil. Dilediğim yerde başka görünüşler seçebilmeliyim. 

Evet. Yazıya örk ekleyin diyeceksiniz. Bunu seziyorum. Gelecekte sayfanın başlığını değiştirdiğimde bu yazılar değişmeyecek. Dinamik olmadığı için bütün link yazılarını bulup güncellemem gerekecek. Kullanışlı değil. 

![resim](https://user-images.githubusercontent.com/25375593/215233790-6ab90bec-7b90-4fd8-8b67-e8d11209d811.png)

### What's your desired outcome?

If this idea is added, I can add the links of many documents between the articles without hesitation. Thanks to this idea, I won't have to constantly edit the posts when there is a change.

---

Bu fikir eklenir ise birçok belgenin örkünü yazı arasına kaygısızca ekleyebilirim. Bu fikir sayesinde bir değişiklik olduğunda yazıları sürekli düzenlemek zorunda kalmayacağım. 

### What's the impact of this for you?

- [X] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by isle9** [comment]
Are there any plans on implementing this? Adding many page links is not an option since it look way to cluttered because of all the icons.

**Comment by taszap** [comment]
Hi both,

I might be missing something but this is already possible. 
When you create a link and add it through the menu that opens, this link shouldn't break if the page is moved. 

<img width="445" alt="image" src="https://github.com/GitbookIO/community/assets/63643069/588d2cec-8430-4502-959c-c6eda55fd726">

**Comment by volkantash** [comment]
Hi. Yes. You're missing something.

Let's say we created a title. Then we added this as a link. The title we created appears where we added it. When the time came, we changed the title. The title changes here too.

If we select a text and add it to it, the text will not change when the title changes. That's why the feature with icon at the top should be without icon whenever we want.

---
Selam. Evet. Bir şeyleri kaçırıyorsunuz.

Biz bir başlık oluşturduk diyelim. Sonra bunu örk olarak ekledik. Eklediğimiz yerde oluşturduğumuz başlık görünüyor. Yeri geldi biz başlığı değiştirdik. Burada da başlık değişiyor. 

Biz bunu bir yazıyı seçip üstüne ekler isek başlık değiştiğinde yazı değişmeyecek. Bu yüzden üstteki simgeli özelliğin istediğimiz zaman simgesizi olmalı.

**Reply by taszap** [reply]
Thank you for clarifying! 
I don't think this is something we are planning in the near future but I will share it with our team as a future improvement. 
We will make sure to update you if it has been accepted into our backlog.
